mlbench_core/utils/pytorch/__init__.py

```diff
@@ -34,6 +37,14 @@ def initialize_backends(
     """
 
     if not (hasattr(dist, "_initialized") and dist._initialized):
+        
+        if comm_backend != "mpi":
```

Comment:
I think we should explicitly list the backends instead of excluding mpi because it would be more understandable for future use and maintenance. So, I suggest we should use `if comm_backend == "gloo" or comm_backend == "nccl"` here.

Reply:
True, cause we might add new backends in the future